docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfiguration.java

```diff
@@ -27,10 +29,14 @@
     public static final String BOOT2DOCKER_PATH = "boot2dockerPath";
     public static final String DOCKER_MACHINE_PATH = "dockerMachinePath";
     public static final String DOCKER_MACHINE_NAME = "machineName";
+    public static final String DOCKER_MACHINE_VERSION = "machineVersion";
+    public static final String DOCKER_MACHINE_DRIVER = "machineDriver";
     private static final String AUTO_START_CONTAINERS = "autoStartContainers";
     private static final String DEFINITION_FORMAT = "definitionFormat";
     static final String DIND_RESOLUTION = "dockerInsideDockerResolution";
     private static final String CUBE_ENVIRONMENT = "cube.environment";
+    private static final String DOCKER_MACHINE_ARQUILLIAN_PATH = "~/.arquillian/machine";
```

Comment:
if it is possible, please make this configurable as well so I can put my downloaded machines wherever I want